05/components/Menu.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import MenuItem from "./MenuItem";
+
+const Menu = () => {
+  return (
+    <ul>
+      <MenuItem text="home" url="/" />
+      <MenuItem text="kontakt" url="/contact" />
```

Comment:
Spróbuj jeszcze tak zrobić, aby przez `props` przekazać tablicę i za jej pomocą i wykorzystaniu `map()` wygenerować elementy menu ;)

Reply:
Właśnie zastanawiałem się jak to ostatecznie zrobić i ostatecznie coś takiego mi wyszło. Nie wiem czy dobrze :D